app/views/pages/show.slim

```diff
@@ -1,4 +1,7 @@
 - meta optimization_tags: @page.meta_tags
+
+/Twitter Meta Tags for proper behavior there
+- meta twitter: {card: 'summary_large_image', domain: 'http://sumofus.org', site: @page.meta_twitter_handle, creator: @page.meta_twitter_handle, title: @page.title, description: truncate(@page.content, length: 140), img: {src: @page.primary_image.try(:content).try(:url)}}
```

Comment:
```
- meta_twitter_handle = t('share.twitter_handle') # I don't think you needs to pass locale: @page.language.code because I'm pretty sure the language is set in this view render
- meta twitter: { site: meta_twitter_handle, creator: meta_twitter_handle, .... 
```